ipapython/ipautil.py

```diff
@@ -245,6 +245,21 @@ def set_ip_net(self, ifnet):
         self._net = ifnet
 
 
+class CheckedIPAddressLoopback(CheckedIPAddress):
+    """IPv4 or IPv6 address with additional constraints with
+    possibility to use a loopback IP.
+    Reserved or link-local addresses are never accepted.
+    """
+    def __init__(self, addr, parse_netmask=True, allow_multicast=False):
+
+        super(CheckedIPAddressLoopback, self).__init__(
+                addr, parse_netmask=parse_netmask, allow_loopback=True,
```

Comment:
Actually, I would like to remove the parameters from `__init__` and have something like: 
```python
    def __init__(self, addr):

        super(CheckedIPAddressLoopback, self).__init__(
                addr, 
                parse_netmask=parse_netmask, 
                allow_multicast=allow_multicast,
                allow_loopback=True)
```

Would this break something?